modular_skyrat/modules/nsfw/vendors/code/naughty_vendor.dm

```diff
@@ -1,7 +1,7 @@
 /obj/machinery/vending/dorms
-	name = "\improper Dorms-Time Vending Machine"
-	desc = "A vending machine for all your fun games and activities in the recreational room!"
-	product_ads = "Hey you, yeah you... wanna take a look at my collection?;Come on, take a look!;Remember, always adhere to Nanotrasen corporate policy!"
+	name = "KinkMate"
+	desc = "A vending machine for all your unmentionable desires."
+	product_ads = "Hey you, yeah you... wanna take a look at my collection~?;Kinky!;Sexy!;Check me out, big boy~"
```

Comment:
?~ is more visually appealing than ~?, In my opinion. A minor thing, but this is a minor change.

Reply:
Wasn't sure myself. Any case, changed.